maven/bnd-indexer-maven-plugin/src/main/java/aQute/bnd/maven/indexer/plugin/IndexerMojo.java

```diff
@@ -83,6 +83,9 @@
 	@Parameter(property = "bnd.indexer.includeTransitive", defaultValue = "true")
 	private boolean						includeTransitive;
 
+	@Parameter(property = "bnd.indexer.includeCurrentIfJar", defaultValue = "false")
+	private boolean						includeCurrentIfJar;
```

Comment:
Can we just call it `includeJar`? This means, if the project makes a jar, that jar is included in the index.